build.sh

```diff
@@ -28,4 +28,8 @@ run_as_root_user ln -s /usr/lib64/libbpf.so.1 /usr/lib/x86_64-linux-gnu/libbpf.s
 run_as_root_user ldconfig
 
 # Run unit tests
-LD_LIBRARY_PATH="${PREFIX_DIR}/usr/local/lib:/usr/local/lib" "${BUILD_DIR}/bin/mcm_unit_tests"
+export LD_LIBRARY_PATH="${PREFIX_DIR}/usr/local/lib:/usr/local/lib"
+"${BUILD_DIR}/bin/sdk_unit_tests"
+"${BUILD_DIR}/bin/media_proxy_unit_tests"
+
+echo -e "\e[48;2;0;255;0;30m Build Succeeded \e[0m"
```

Comment:
Could you change the build.sh in next PR and just use one of methods that are implemented and included in the scriipt:
 
![image](https://github.com/user-attachments/assets/36f375b6-55f2-45d1-a764-4ee1162f25a6)
 
Instead of this .
 
you can just write: prompt Build Succeeded

![image](https://github.com/user-attachments/assets/2160ec15-e4a8-49ef-af8a-34842a7cf405)
 
it will be blue but I can add the succeeded method with proper colors - the above can have be disabled here: https://github.com/OpenVisualCloud/Media-Communications-Mesh/blob/main/scripts/common.sh
 
so that it can be done globally for placess where it should

Reply:
Good idea! Will do in the next PR.

Reply:
Fixed in https://github.com/OpenVisualCloud/Media-Communications-Mesh/commit/1f11bbc0e781d91348d91e782c09d1d35e2c87ee